### PR TITLE
CI with Java 15

### PR DESCRIPTION
**Pull request by 3flex**
14 is EOL

### CODE REVIEW

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UseIfEmptyOrIfBlank.kt [hunk 1]
```diff
@@ -127,7 +127,8 @@ class UseIfEmptyOrIfBlank(config: Config = Config.empty) : Rule(config) {
             Replacement(FqName("kotlin.collections.Collection.isEmpty"), ifEmpty),
             Replacement(FqName("kotlin.text.isNotBlank"), ifBlank, negativeCondition = true),
             Replacement(FqName("kotlin.text.isNotEmpty"), ifEmpty, negativeCondition = true),
-            Replacement(FqName("kotlin.collections.isNotEmpty"), ifEmpty, negativeCondition = true)
+            Replacement(FqName("kotlin.collections.isNotEmpty"), ifEmpty, negativeCondition = true),
+            Replacement(FqName("kotlin.String.isEmpty"), ifEmpty)
```
Comment (BraisGabin) [inline detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UseIfEmptyOrIfBlank.kt:131]: Could you move this to another pr? And add a test so we don't have the same error again.

**Review by BraisGabin — APPROVED** [review]

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UseIfEmptyOrIfBlank.kt (hunk 1 above)
Comment (3flex) [reply detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UseIfEmptyOrIfBlank.kt:131]: I don't think that makes sense because I had to add this to get the build passing with Java 15 in the first place. So that new line is already effectively being tested by CI on Java 15.
